simulariumio/file_converter.py

```diff
@@ -34,53 +32,78 @@ def __init__(self, input_path: str):
             buffer_data = json.load(simularium_file)
         if (
             int(buffer_data["trajectoryInfo"]["version"])
-            < self.current_trajectory_info_version
+            < CURRENT_VERSION.TRAJECTORY_INFO
         ):
             buffer_data = self.update_trajectory_info_version(buffer_data)
         self._data = TrajectoryData.from_buffer_data(buffer_data)
 
-    def update_trajectory_info_version(self, data: Dict[str, Any]) -> Dict[str, Any]:
+    @staticmethod
+    def update_trajectory_info_v1_to_v2(data: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Update the trajectory info block from v1 to v2
```

Comment:
add a docstring comment saying that this mutates the object passed in, and returns it.  it does not return a new copy

Reply:
also, why not pass in data["trajectoryInfo"] instead?  will these functions ever modify any other part of `data`?